presto-main/src/main/java/com/facebook/presto/sql/relational/CallExpression.java

```diff
@@ -52,6 +52,12 @@ public Type getType()
         return returnType;
     }
 
+    @Override
+    public boolean isDeterministic()
+    {
+        return DeterminismEvaluator.isDeterministic(this);
```

Comment:
This is weird to add an abstract method that always calls a static function. Why not just use the static function directly?